.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "db-snapshot"]
 	path = db-snapshot
 	url = ../db-snapshot
+[submodule "third_party/openmrs-module-sync"]
+	path = third_party/openmrs-module-sync
+	url = git@github.com:openmrs/openmrs-module-sync.git
```

Comment:
@capnfabs:
What is the advantage of creating this as a git submodule? openmrs-module-xforms is not treated the same.
Shouldn't we avoid breaking things with OpenMRS updatint the module?
Can't we copy the version we are using like we did for xforms?